vrk-kpa/suomidigi
# SUOPA-792 Blog posts

To test: 
- `make fresh drush-cr`
- Log in and go to edit the front page. https://suomidigi.docker.sh/node/7/edit
   - Scroll down to "Tuoreimmat blogikirjoitukset"
   - Edit each of the liftups and make sure the "full width" checkboxes are NOT checked
   - Save the page and check that the blog post liftups looks ok.
- Go to https://suomidigi.docker.sh/node/405/edit
   - Edit the View paragraph and select "Ruudukko" for the view display
   - Save the page and check that the blog post teasers looks like the ones in proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/49937afb-7511-44c7-8bef-db8bb61686f6/Suomidigi-2020-blogi-XL-1
